GliaWidgets/Public/Glia/Glia+StartEngagement.swift

```diff
@@ -28,20 +28,25 @@ extension Glia {
         // `GliaError.engagementExists` is no longer thrown,
         // instead engagement is getting restored.
         guard let configuration = self.configuration else { throw GliaError.sdkIsNotConfigured }
-
+        self.interactor?.setQueuesIds(queueIds)
```

Comment:
I think comment here would be useful why we set queue IDs. Also about the fact that interaction is optional.

Reply:
Fixed [e8c7c5a](https://github.com/salemove/ios-sdk-widgets/pull/1011/commits/e8c7c5a43a0ccb65a53bc07c4403c8206f003453)